sdk/azcore/runtime/pipeline.go

```diff
@@ -34,8 +34,14 @@ type PipelineOptions struct {
 	// Each policy is executed once per request, and for each retry of that request.
 	PerRetry []policy.Policy
 
-	// TracingNamespace contains the value to use for the az.namespace span attribute.
-	TracingNamespace string
+	// Tracing contains options used to configure distributed tracing.
+	Tracing TracingOptions
+}
+
+// TracingOptions contains tracing options for SDK developers.
+type TracingOptions struct {
```

Comment:
@JeffreyRichter we don't _have_ to do this but it seems logical to group any future tracing options for the pipeline together.